hr_employee_medical_examination/models/hr_employee_medical_examination.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployeeMedicalExamination(models.Model):
+
+    _name = 'hr.employee.medical.examination'
+    _description = 'Hr Employee Medical Examination'
+    _inherit = ['mail.thread']
```

Comment:
Why not activity? Maybe it is interesting to make a relation to an activity / reminder